src/Native/configure.cmake

```diff
@@ -270,6 +270,13 @@ check_function_exists(
     inotify_rm_watch
     HAVE_INOTIFY_RM_WATCH)
 
+check_cxx_source_compiles(
```

Comment:
Unfortunately, no.  From the cmake docs:
```
If the symbol is a type or enum value it will not be recognized (consider using CheckTypeSize or CheckCSourceCompiles).
```